<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatternSelector</class>
 <widget class="QWidget" name="PatternSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PatternSelector</string>
  </property>
  <property name="styleSheet">
   <string>QGroupBox { font-weight: bold; }</string>
  </property>
  <widget class="QLabel" name="Index">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="PatternColorComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="PatternColorLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>图案颜色</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>409</width>
      <height>89</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>411</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="PatternStyleGLayout"/>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="PatternColorLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>图案样式</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
